src/transformState.js

```diff
@@ -62,7 +62,27 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (let i = 0; i < transforms.length; i++) {
+    if (transforms[i].operation === 'addProperties') {
+      for (const prop in transforms[i].properties) {
+        state[prop] = transforms[i].properties[prop];
+      }
+    }
+
+    if (transforms[i].operation === 'removeProperties') {
+      for (const prop of transforms[i].properties) {
```

Comment:
for (const property of transform.properties) {

Reply:
Do you mean not to use an abbreviation?

Reply:
It's done anyway.